website/src/pages/BlogPage/index.tsx

```diff
@@ -10,6 +10,7 @@ import { useViewport } from "../../hooks/viewportWidth";
 import { SOCIAL_PLATFORMS, VIEW_PORT } from "../../constants";
 import BlogsSlider from "../../components/BlogsSlider";
 import Newsletter from "../../components/Newsletter";
+import { SlackShareIcon } from "./slack-share-icon";
```

Comment:
we can use pascal casing here `slack-share-icon` -> `SlackShareIcon`

Reply:
fair point @rakeshPRaghu  -- in hooks, we are using camelCase such as `useExternalStyles,  useQuery` and CommercialSupport  page also. 

Changing to `camelCase` for consistency since `SlackShareIcon` is a file not folder :)